test_runner/src/main/kotlin/ftl/args/yml/errors/ConfigurationErrorParser.kt

```diff
@@ -0,0 +1,30 @@
+package ftl.args.yml.errors
+
+internal class ConfigurationErrorParser {
```

Comment:
no state, no constructor args, no needs for class.

Reply:
Thanks for this! I made changes :)